main/src/library/Concurrent/Condition.flix

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 Jonathan Lindegaard Starup
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Concurrent {
+
+    ///
+    /// A wrapper around a Java Condition.
+    ///
+    opaque type Condition = ##java.util.concurrent.locks.Condition
```

Comment:
I think this should be public? It is used as an argument type for public methods but I'm not sure

Reply:
Should this be placed inside the Condition namespace?